### ISSUE TITLE
Feature request: Toggle RGB on/off

### ISSUE THREAD

**Issue by DarrenVs** [WootingKb/wootility-issues#84]
### Describe the feature request
It would be nice to have a fast way to toggle RGB on/off instead of holding down the brightness buttons.
## 

### Concept
![image](https://user-images.githubusercontent.com/2780639/107133076-fbb97680-68e4-11eb-952e-56b9753d009f.png)
![image](https://user-images.githubusercontent.com/2780639/107133077-fc520d00-68e4-11eb-84f1-9eb6aa68c438.png)

**Comment by DarrenVs** [comment]
Seems like the Scroll-Lock key on the Wooting Two/He/Lekker has room for a function like this

**Comment by simon-wh** [comment]
I've got a question about how this function should behave.

Would it be like a volume mute button where when you "unmute" it goes back to the brightness setting you were already on? Or would it behave like just toggling the brightness value between 0 & 100?

**Comment by DarrenVs** [comment]
I'd say a toggle between 0% and original profile brightness %

**Comment by denis-roy** [comment]
Thanks @DarrenVs for initiating the issue and @simon-wh for looking into this

+1 for the described workflow of mute = 0%, unmute = previous

**Comment by Kallestofeles** [comment]
I would like to vouch my vote for this feature as I too am dimming the backlight on a daily basis. Will mod the 60HE soon to not have a bright plate beneath the keys, but having this feature would be a really nice boon, not to mention, it should not be a difficult implementation as all the building blocks are already there.

**Comment by T3h-vICE** [comment]
Edit: i found a workaround for this problem. I'll explain step by step.
1. Open Wootility
2. Clone the profile, where you want to turn off the lights with a key. Mine is Typing and i renamed the clone into Typing LED off.
3. Replace your cloned profile with another profile that you never use anyways, so it ends up as the last profile (3).
4. Go to the color options and paint all your keys black. Only for the FN key and the main profile i used purple colors, so i can easily turn on RGB in the dark, without hitting the wrong keys.
5. Make sure you saved every profile.
6. I'm serious! Make sure you saved every profile.
7. now you have a key that turns the LEDs off, by simply switching the profile, which is just a clone with different color settings

> I was also searching to find this feature. It's so confusing, that there's no shortcut to turn it off instantly. In fact i've never seen a RGB keyboard, without a rgb toggle button.
> Well since it's over 3 years now, seems like we will never get that toggle.

**Comment by kodoz** [comment]
Yeah, toggle RGB action is necessary and very handy, like the one in the VIA app for custom keebs
